airbyte-integrations/connectors/destination-mysql/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'application'
+    id 'airbyte-docker'
+    id 'airbyte-integration-test-java'
+}
+
+application {
+    mainClass = 'io.airbyte.integrations.destination.mysql.MySQLDestination'
+}
+
+dependencies {
+    implementation project(':airbyte-db')
+    implementation project(':airbyte-integrations:bases:base-java')
+    implementation project(':airbyte-protocol:models')
+    implementation project(':airbyte-integrations:connectors:destination-jdbc')
+
+    // https://mvnrepository.com/artifact/mysql/mysql-connector-java
```

Comment:
nit: not really necessary to include this since all of our public mvn dependencies can be looked up the same way